website/docs/deployment.mdx

```diff
@@ -140,14 +140,172 @@ cmd /C 'set "GIT_USER=<GITHUB_USERNAME>" && yarn deploy'
+
+      # Installs v14 LTS of Node:
+      # (Uses cached version that Github replaces bi-weekly with latest updates for that version of Node)
+      # Docs: https://github.com/marketplace/actions/setup-node-js-environment
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+
+      - name: Get yarn cache
+        id: yarn-cache
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      # Cache the yarn cache dir to speed up builds if dependencies haven't changed:
+      # (Caches are deleted if they've not been used in the past 7 days,
+      # or the total cache limit of 5GB is exceeded (oldest cache data is evicted first)).
+      # Docs: https://github.com/actions/cache
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.yarn-cache.outputs.dir }}
+          key: ${{ runner.os }}-website-${{ hashFiles('**/yarn.lock') }}
+          # OPTIONAL: If the cache key does not have a match, `restore-keys` will check for
+          # previous cache keys with the given prefix and choose the most recent cache:
+          # https://docs.github.com/en/actions/guides/caching-dependencies-to-speed-up-workflows#matching-a-cache-key
+          restore-keys: ${{ runner.os }}-website-
+
+      # Install project dependencies, then run the build script from `package.json`:
+      # (Outputs static docs for hosting to `website/build/`)
+      - run: yarn install --frozen-lockfile
+      - run: yarn build
+
+      # Popular action to deploy to gh-pages:
+      # Docs: https://github.com/peaceiris/actions-gh-pages
+      - name: 'Deploy to Github Pages'
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          # Generated per workflow run, provides automatic auth to push:
+          # Docs: https://docs.github.com/en/actions/reference/authentication-in-a-workflow#about-the-github_token-secret
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          # Build directory contents to publish to the `gh-pages` branch:
+          publish_dir: ./website/build
+          # OPTIONAL: Directory to place `publish_dir` contents on the `gh-pages` branch:
+          # (Defaults to root of project, removes contents at target dir by default)
+          # destination_dir: ${{ env.DOCS_VERSION }}
+          user_name: ${{ env.GIT_USER }}
+          user_email: ${{ env.GIT_EMAIL }}
+```
+
+While you can have both jobs defined in the same workflow file, the deploy job as configured in the `remote` example will always be listed as skipped in the PR check suite status.
+
+That's added noise providing no value to the review process, and as you cannot easily share common snippets like the build process; it is better to manage as separate workflows instead:
+
+```yaml title=".github/workflows/documentation-pr.yml"
+name: 'Docs (PR)'
+
+on:
+  pull:
+    branches:
+      - documentation
+    paths:
+      - 'website/**'
```

Comment:
I kept this config example for pull requests more terse since the prior config is full of related comments explaining how the workflow works.

As this uses the `on.pull` event trigger instead, it might be worth clarifying that this is related to PRs made targeting the `documentation` branch to merge to? (not PRs where the remote has the branch `documentation`)